### PR TITLE
Add tslint configurations

### PR DESCRIPTION
**Pull request by sponkrashin**

### CODE REVIEW

File: Arcadia.Ask/ClientApp/src/app/questions/questions-store.service.ts [hunk 1]
```diff
@@ -28,7 +26,7 @@ export class QuestionsStore implements OnDestroy {
     this.questionsSubscription = (this.questions as ConnectableObservable<Map<string, Question>>).connect();
   }
 
-  public createQuestion(text: string) {
+  public async createQuestion(text: string) {
```
Comment (santee) [inline Arcadia.Ask/ClientApp/src/app/questions/questions-store.service.ts]: no

Comment (sponkrashin) [reply Arcadia.Ask/ClientApp/src/app/questions/questions-store.service.ts]: Why not? All other similar methods in this class have `async` word.

Comment (sponkrashin) [reply Arcadia.Ask/ClientApp/src/app/questions/questions-store.service.ts]: All we should remove those async's too.

Comment (santee) [reply Arcadia.Ask/ClientApp/src/app/questions/questions-store.service.ts]: yeah, missed that. those should not have `async` either. It generates all kinds of weird statements, just to return a promise, and we already do that.

Comment (sponkrashin) [reply Arcadia.Ask/ClientApp/src/app/questions/questions-store.service.ts]: Removed.

**Pull request merged**